pyvista/plotting/charts.py

```diff
@@ -196,11 +196,14 @@ def color(self):  # numpydoc ignore=RT01
         --------
         Set the pen's color to red.
 
-        >>> import pyvista as pv
-        >>> chart = pv.Chart2D()
-        >>> plot = chart.line([0, 1, 2], [2, 1, 3])
-        >>> plot.pen.color = 'r'
-        >>> chart.show()
+        .. pyvista-plot::
+           :force_static:
```

Comment:
I wonder if there is a way to configure `:force_static:` at the module level.... looks okay though!

Reply:
It could be possible but if its okay with you I will  look into that in a separate MR. I will add it though to the list of issues (along with the rest of the checkboxes in the PR description.) once this is merged.

Reply:
It could be possible but if its okay with you I will look into that in a separate MR. I will add it though to the list of issues (along with the rest of the checkboxes in the PR description.) once this is merged.

Reply:
Yeah, good by me to label as technical debt to come back to